### ISSUE TITLE
Regenerating helptags error after last update

### ISSUE THREAD

**Issue by unclechu** [Quramy/tsuquyomi#157]
Last lines from log after `:PluginUpdate` provided by [Vundle](https://github.com/VundleVim/Vundle.vim):
```text
[2017-05-28 00:23:12] Helptags:
[2017-05-28 00:23:12] :helptags /home/unclechu/.config/git-repos/neovimrc/bundle/Vundle.vim/doc
[2017-05-28 00:23:12] :helptags /home/unclechu/.config/git-repos/neovimrc/bundle/nerdtree/doc
[2017-05-28 00:23:12] :helptags /home/unclechu/.config/git-repos/neovimrc/bundle/nerdcommenter/doc
[2017-05-28 00:23:12] :helptags /home/unclechu/.config/git-repos/neovimrc/bundle/editorconfig-vim/doc
[2017-05-28 00:23:12] :helptags /home/unclechu/.config/git-repos/neovimrc/bundle/vim-easy-align/doc
[2017-05-28 00:23:12] :helptags /home/unclechu/.config/git-repos/neovimrc/bundle/vim-indexed-search/doc
[2017-05-28 00:23:12] :helptags /home/unclechu/.config/git-repos/neovimrc/bundle/vim-multiple-cursors/doc
[2017-05-28 00:23:12] :helptags /home/unclechu/.config/git-repos/neovimrc/bundle/vim-indent-guides/doc
[2017-05-28 00:23:12] :helptags /home/unclechu/.config/git-repos/neovimrc/bundle/tagbar/doc
[2017-05-28 00:23:12] :helptags /home/unclechu/.config/git-repos/neovimrc/bundle/vim-airline/doc
[2017-05-28 00:23:12] :helptags /home/unclechu/.config/git-repos/neovimrc/bundle/vim-airline-themes/doc
[2017-05-28 00:23:12] :helptags /home/unclechu/.config/git-repos/neovimrc/bundle/vim-ctrlspace/doc
[2017-05-28 00:23:12] :helptags /home/unclechu/.config/git-repos/neovimrc/bundle/my-ultisnips/doc
[2017-05-28 00:23:12] :helptags /home/unclechu/.config/git-repos/neovimrc/bundle/incsearch.vim/doc
[2017-05-28 00:23:12] :helptags /home/unclechu/.config/git-repos/neovimrc/bundle/vim-commentary/doc
[2017-05-28 00:23:12] :helptags /home/unclechu/.config/git-repos/neovimrc/bundle/unite.vim/doc
[2017-05-28 00:23:12] :helptags /home/unclechu/.config/git-repos/neovimrc/bundle/neomru.vim/doc
[2017-05-28 00:23:12] :helptags /home/unclechu/.config/git-repos/neovimrc/bundle/neoyank.vim/doc
[2017-05-28 00:23:12] :helptags /home/unclechu/.config/git-repos/neovimrc/bundle/deoplete.nvim/doc
[2017-05-28 00:23:12] :helptags /home/unclechu/.config/git-repos/neovimrc/bundle/gundo.vim/doc
[2017-05-28 00:23:12] :helptags /home/unclechu/.config/git-repos/neovimrc/bundle/vim-startify/doc
[2017-05-28 00:23:12] :helptags /home/unclechu/.config/git-repos/neovimrc/bundle/ack.vim/doc
[2017-05-28 00:23:12] :helptags /home/unclechu/.config/git-repos/neovimrc/bundle/vim-easymotion/doc
[2017-05-28 00:23:12] :helptags /home/unclechu/.config/git-repos/neovimrc/bundle/vim-move/doc
[2017-05-28 00:23:12] :helptags /home/unclechu/.config/git-repos/neovimrc/bundle/delimitmate/doc
[2017-05-28 00:23:12] :helptags /home/unclechu/.config/git-repos/neovimrc/bundle/ctrlsf.vim/doc
[2017-05-28 00:23:12] :helptags /home/unclechu/.config/git-repos/neovimrc/bundle/vim-quickhl/doc
[2017-05-28 00:23:12] :helptags /home/unclechu/.config/git-repos/neovimrc/bundle/braceless.vim/doc
[2017-05-28 00:23:12] :helptags /home/unclechu/.config/git-repos/neovimrc/bundle/neovim-gui-shim/doc
[2017-05-28 00:23:12] :helptags /home/unclechu/.config/git-repos/neovimrc/bundle/vim-devicons/doc
[2017-05-28 00:23:12] :helptags /home/unclechu/.config/git-repos/neovimrc/bundle/vim-surround/doc
[2017-05-28 00:23:12] :helptags /home/unclechu/.config/git-repos/neovimrc/bundle/neomake/doc
[2017-05-28 00:23:12] :helptags /home/unclechu/.config/git-repos/neovimrc/bundle/vim-fugitive/doc
[2017-05-28 00:23:12] :helptags /home/unclechu/.config/git-repos/neovimrc/bundle/vim-gitgutter/doc
[2017-05-28 00:23:12] :helptags /home/unclechu/.config/git-repos/neovimrc/bundle/ghcmod-vim/doc
[2017-05-28 00:23:12] :helptags /home/unclechu/.config/git-repos/neovimrc/bundle/vim-hoogle/doc
[2017-05-28 00:23:12] :helptags /home/unclechu/.config/git-repos/neovimrc/bundle/vim-faust/doc
[2017-05-28 00:23:12] :helptags /home/unclechu/.config/git-repos/neovimrc/bundle/vim-fireplace/doc
[2017-05-28 00:23:12] :helptags /home/unclechu/.config/git-repos/neovimrc/bundle/vim-flow/doc
[2017-05-28 00:23:12] :helptags /home/unclechu/.config/git-repos/neovimrc/bundle/vim-coffee-script/doc
[2017-05-28 00:23:12] :helptags /home/unclechu/.config/git-repos/neovimrc/bundle/vimproc.vim/doc
[2017-05-28 00:23:12] :helptags /home/unclechu/.config/git-repos/neovimrc/bundle/tsuquyomi/doc
[2017-05-28 00:23:12] > Error running :helptags /home/unclechu/.config/git-repos/neovimrc/bundle/tsuquyomi/doc
[2017-05-28 00:23:12] :helptags /home/unclechu/.config/git-repos/neovimrc/bundle/vim-markdown/doc
[2017-05-28 00:23:12] :helptags /home/unclechu/.config/git-repos/neovimrc/bundle/vim-ansible-yaml/doc
[2017-05-28 00:23:12] :helptags /home/unclechu/.config/git-repos/neovimrc/bundle/emmet-vim/doc
[2017-05-28 00:23:12] Helptags: 45 plugins processed
```
`$MYVIMRC/bundle/tsuquyomi` HEAD is `254f890f2f216a5179d8612f8a2b6fa15ca461ab`

Neovim `:version`:
```
NVIM v0.2.0
Build type: Debug
Compilation: /usr/bin/cc -Wconversion -DNVIM_MSGPACK_HAS_FLOAT32 -g -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wvla -fstack-protector-strong -fdiagnostics-color=auto -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -I/home/unclechu/.compile_n_build/neovim/neovim-0.2.0/build/config -I/home/unclechu/.compile_n_build/neovim/neovim-0.2.0/src -I/home/unclechu/.compile_n_build/neovim/neovim-0.2.0/.deps/usr/include -I/home/unclechu/.compile_n_build/neovim/neovim-0.2.0/.deps/usr/include -I/home/unclechu/.compile_n_build/neovim/neovim-0.2.0/.deps/usr/include -I/home/unclechu/.compile_n_build/neovim/neovim-0.2.0/.deps/usr/include -I/home/unclechu/.compile_n_build/neovim/neovim-0.2.0/.deps/usr/include -I/home/unclechu/.compile_n_build/neovim/neovim-0.2.0/.deps/usr/include -I/usr/include -I/home/unclechu/.compile_n_build/neovim/neovim-0.2.0/build/src/nvim/auto -I/home/unclechu/.compile_n_build/neovim/neovim-0.2.0/build/include
Compiled by unclechu@fedorafarm25

Optional features included (+) or not (-): +acl   +iconv    +jemalloc +tui      
For differences from Vim, see :help vim-differences

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/home/unclechu/neovim-v0.2.0/share/nvim"
```

**Comment by Quramy** [comment]
Sorry, I've fixed it at bc3f5a74bc9a1a5d527dacbe07a8af78d1abcad5 🙇

**Comment by unclechu** [comment]
@Quramy Thanks a lot! Now everything is okay.
